openapi/docs/SourceSnowflake.md

```diff
@@ -0,0 +1,145 @@
+# SourceSnowflake
+
+## Properties
+
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**Database** | **string** | Name of the snowflake database. | 
+**Schema** | **string** | Name of the snowflake database schema. | 
+**TableName** | **string** | Name of the snowflake table. | 
+**Warehouse** | Pointer to **string** | Name of the data warehouse to be used. | [optional] 
+**Status** | Pointer to [**StatusSnowflake**](StatusSnowflake.md) |  | [optional] 
+
+## Methods
+
+### NewSourceSnowflake
+
+`func NewSourceSnowflake(database string, schema string, tableName string, ) *SourceSnowflake`
+
+NewSourceSnowflake instantiates a new SourceSnowflake object
+This constructor will assign default values to properties that have it defined,
+and makes sure properties required by API are set, but the set of arguments
+will change when the set of required properties is changed
+
+### NewSourceSnowflakeWithDefaults
+
+`func NewSourceSnowflakeWithDefaults() *SourceSnowflake`
+
+NewSourceSnowflakeWithDefaults instantiates a new SourceSnowflake object
+This constructor will only assign default values to properties that have it defined,
+but it doesn't guarantee that properties required by API are set
```

Comment:
These lack an ending period, but all others seem to have one.

Reply:
this is auto-generated